package.json

```diff
@@ -41,11 +41,11 @@
     "@salesforce/kit": "^1.0.0",
     "@salesforce/ts-sinon": "^1.0.0",
     "@salesforce/ts-types": "^1.0.0",
-    "@types/jsforce": "1.9.2",
```

Comment:
We won't accept this change to package.json. There is a breaking change in jsforce 1.9.1.

https://github.com/jsforce/jsforce/issues/797

Reply:
@catalandres Can you just update the types and not update the jsforce dependency?